src/StackAdmin/Azs.InfrastructureInsights.Admin/Tests/src/Alert.Tests.ps1

```diff
@@ -225,5 +226,22 @@ InModuleScope Azs.InfrastructureInsights.Admin {
                 }
             }
         }
+
+        it "TestRepairAlert" -Skip:$('TestRepairAlert' -in $global:SkippedTests) {
```

Comment:
You should add a test for alerts that cannot be repaired and repairs on alerts that don't even exist.

Reply:
No, we use these tests in VaaS for validation.  These are required.